docs/source/plugin-directory.rst

```diff
@@ -12,10 +12,11 @@ contains in parentheses:
 * `intake-accumulo <https://github.com/intake/intake-accumulo>`_ Apache Accumulo clustered data storage (``accumulo``)
 * `intake-avro <https://github.com/intake/intake-avro>`_: Apache Avro data serialization format (``avro_table``, ``avro_sequence``
 * `intake-bluesky <https://nsls-ii.github.io/intake-bluesky/>`_: search and retrieve data in the `bluesky <https://nsls-ii.github.io/bluesky>`_ data model
+* `intake-dcat <https://github.com/CityOfLosAngeles/intake-dcat>`_ Browse and load data from `DCAT <https://www.w3.org/TR/vocab-dcat>`_ catalogs.
```

Comment:
Does the DCAT catalog driver have a driver name of it's own?

Reply:
Not at the moment, no. On the todo list, but right now I suspect it does not function well as a source for other catalogs.

Reply:
ok